observability/otel/otel-rn.adoc

```diff
@@ -27,11 +116,7 @@ The {OTELName} 3.5.1 is based on the open source link:https://opentelemetry.io/d
 
 This release fixes the following CVEs:
 
-* link:https://access.redhat.com/security/cve/CVE-2025-22868[CVE-2025-22868]
-* link:https://access.redhat.com/security/cve/CVE-2025-27144[CVE-2025-27144]
-* link:https://access.redhat.com/security/cve/CVE-2025-29786[CVE-2025-29786]
-* link:https://access.redhat.com/security/cve/CVE-2025-30204[CVE-2025-30204]
-
+* 
```

Comment:
```suggestion
```
I would comment this all out if there aren't any. Easy to merge that and forget.